net-wireless/blueman/blueman-2.3.1.ebuild

```diff
@@ -97,7 +97,7 @@ pkg_setup() {
 }
 
 src_prepare() {
-	[[ ${PV} == 9999 ]] && eautoreconf
+	[[ ${PV} == 9999 ]] && eautoreconf || eautomake
```

Comment:
Note that this can be dropped on next release I think: https://github.com/blueman-project/blueman/commit/2e2e43bce5f22c1256f1e333a1f11feb20271a49